### DISCUSSION TITLE
Hi, its been long time can not fix this warning , anyone else have experiences with this warning ? #expo #reactnative

### DISCUSSION THREAD

**Discussion by YahefuYusufu** [EddieHubCommunity/support#7859] [Q&A]
WARN  Constants.platform.ios.model has been deprecated in favor of expo-device's Device.modelName property. This API will be removed in SDK 45.

**Comment by Pradumnasaraf** [comment]
Check out this, I find a GitHub issue on this. https://github.com/expo/expo/issues/15110.

**Comment by YahefuYusufu** [comment]
of course ,appreciate your help

**Reply by Pradumnasaraf** [reply]
Done. Thanks for replying. :)

**Reply by YahefuYusufu** [reply]
case already closed seems like i can not change it , sorry
